src/Playwright/Transport/Converters/EvaluateArgumentValueConverter.cs

```diff
@@ -123,6 +123,19 @@ internal static object Serialize(object value, List<EvaluateArgumentGuidElement>
             return new { bi = bigInteger.ToString(CultureInfo.InvariantCulture) };
         }
 
+        if (value is Exception exception)
+        {
+            return new Dictionary<string, object>
+            {
+                ["e"] = new Dictionary<string, object>
```

Comment:
What's the point in serializing .net error to a js error? I'd not add this at all, especially given that we not parse the error from js and just return a string.